File: src/main/java/pl/mjedynak/idea/plugins/builder/factory/GoToBuilderPopupListFactory.java
package pl.mjedynak.idea.plugins.builder.factory;

import static java.util.Collections.singletonList;

import com.intellij.ui.components.JBList;
import org.jetbrains.annotations.NotNull;
import pl.mjedynak.idea.plugins.builder.action.GenerateBuilderAdditionalAction;

public class GoToBuilderPopupListFactory extends AbstractPopupListFactory {
    @Override
    protected @NotNull JBList<GenerateBuilderAdditionalAction> createList() {
        return new JBList<>(singletonList(new GenerateBuilderAdditionalAction()));
    }
}

File: src/main/kotlin/io/github/kingg22/kobra/builder/factory/GoToBuilderPopupListFactory.kt
package io.github.kingg22.kobra.builder.factory

import com.intellij.ui.components.JBList
import io.github.kingg22.kobra.builder.action.GenerateBuilderAdditionalAction

public class GoToBuilderPopupListFactory : AbstractPopupListFactory() {
    override fun createList(): JBList<GenerateBuilderAdditionalAction> = JBList(GenerateBuilderAdditionalAction())
}
